### DISCUSSION TITLE
Please teach me: How to add Supplier Pricing!

### DISCUSSION THREAD

**Discussion by zly20129** [inventree/InvenTree#7750] [Q&A]
I can't really find how to add supplier prices in the web interface or in the help documentation, could someone who knows how to add it please show me how to add it, thank you very much!

**Comment by zly20129** [comment]
~I can't find how to add Internal Pricing also.~
I find it!

**Comment by SchrodingersGat** [comment]
### Internal Pricing

https://docs.inventree.org/en/stable/part/pricing/#internal-pricing

### Supplier Pricing

Supplier pricing is added via the supplier part detail page:

https://docs.inventree.org/en/stable/order/company/#supplier-parts

An overview is also available in the "part pricing" view.

**Reply by zly20129** [reply]
That is to say: for example, if there are 3 suppliers for part A, and I add part A to all 3 suppliers, and all 3 suppliers have added purchase orders related to part A, the supplier price will be generated automatically without me having to fill it in manually, is that right?
也就是说：比如A部件有3个供应商，我在3个供应商里面都添加A部件，并且3个供应商都添加过A部件相关的采购订单，供应商价格就会自动生成而不需要我手动填写，是这样吗？

**Reply by SchrodingersGat** [reply]
Correct, part pricing will be automatically calculated from *completed* purchase orders

**Reply by zly20129** [reply]
Ok, I get it, thank you very much for the answer!
